packages/minter-contracts/ligo/src/fixed_price_sale/fixed_price_sale_market.mligo

```diff
@@ -1,5 +1,5 @@
 #include "../../fa2/fa2_interface.mligo"
-#include "../../fa2_modules/pauseable_admin_option.mligo"
+#include "../../fa2_modules/admin/simple_admin.mligo"
```

Comment:
The way it is done here, we only get `simple_admin` version of the admin modules. We can restructure it easily so that it will be very easy to get all of the admin module versions. See what is done for example in https://github.com/tqtezos/minter-sdk/blob/main/packages/minter-contracts/ligo/src/minter_collection/nft/fa2_multi_nft_asset_multi_admin.mligo
based on the guidance given in https://github.com/tqtezos/minter-sdk/blob/main/packages/minter-contracts/ligo/fa2_modules/README.md#how-to-compose-reusable-modules-into-a-single-contract

Reply:
For this PR I reverted back to pauseable admin option. The switch to simple admin will come in another PR that also required changing some haskell tests.